front/src/modules/trainschedule/components/ManageTrainSchedule/Map.tsx

```diff
@@ -333,11 +335,12 @@ const Map = ({
           showStdcmAssets={showStdcmAssets}
           isFeasible={isFeasible}
         />
-        {mapRef.current && (
+        {mapRef.current && infraID && (
```

Comment:
We don't need it anymore (same in NewMap)
```suggestion
        {infraID && (
```

Reply:
Done